packages/react-native-web/src/exports/Image/__tests__/index-test.js

```diff
@@ -319,7 +380,7 @@ describe('components/Image', () => {
       );
       expect(container.firstChild).toMatchSnapshot();
       act(() => {
-        loadCallback();
+        loadCallback({ uri });
```

Comment:
First time looking at these tests and probably just having trouble working it out - but is there a reason for passing the `source` here? Doesn't seem like it was needed before.

Reply:
After the changes introduced in this PR it is needed, because the `uri` used under the hood might need to change 

There's logic that uses `URL.createObjectURL` that changes the `uri` with a local blob URL for images that were loaded using `fetch` and `headers` 
This is because the native `image.src` does not support headers and the only way is to make a request (using `fetch)) then create a local URL with the result of the request